packages/field/test/InteractionStateMixin.test.js

```diff
@@ -51,15 +51,15 @@ describe('InteractionStateMixin', async () => {
     // expect(formElement.touched).to.equal(true);
   });
 
-  it('sets a class "state-(touched|dirty)"', async () => {
+  it('sets an attribute "state-(touched|dirty)"', async () => {
```

Comment:
are we going to make it a major version bump? shall we maybe stick to half-year major updates cycle already due to the need to keep extensions of lion stable as well?

Reply:
It would indeed be a breaking change. In order to move on with this pr, we could consider reflecting everything still to classes in the updated callback (so that we can throw away that code in next release).

Reply:
let's ask first if that compatibility is needed as I guess it's only used in our styling and people should not have deviated from it (let's hope)

if needed that compatibility should be on a different layer 😉

Reply:
> if needed that compatibility should be on a different layer 😉
Ah, that's a good idea indeed

Reply:
agreed